docs/virtual_machines/istio_service_mesh.md

```diff
@@ -0,0 +1,147 @@
+# Istio service mesh
+
+Service mesh allows to monitor, visualize and control traffic between pods.
+Kubevirt supports running VMs as a part of Istio service mesh.
+
+## Limitations
+
+- Istio service mesh is only supported with a pod network masquerade binding.
+
+- Istio uses a [list of ports](https://istio.io/latest/docs/ops/deployment/requirements/#ports-used-by-istio) for its own purposes, these ports must not be explicitly specified in a VMI interface.
+
+- Istio only supports IPv4
+
+## Prerequisites
+
+- This guide assumes that Istio is already deployed and uses Istio CNI Plugin. See [Istio documentation](https://istio.io/latest/docs/) for more information.
+
+- `Istioctl` binary is installed. See Istio [installation inctructions](https://istio.io/latest/docs/setup/getting-started/).
```

Comment:
nit: this is only required for debugging / troubleshooting. If you have the chance, add that tiny update here.